### DISCUSSION TITLE
How to use tailwindMergeConfig after creating the object.

### DISCUSSION THREAD

**Discussion by DevYemi** [dcastil/tailwind-merge#417] [Help]
Hi there, thank you for the awesome utility library.

I am trying to fix an issue while using tw-merge where `text-font-styling` cancels `text-colors`.

I have seen in the docs how to fix this with `tailwindMergeConfig ` but the docs do not say anything about where to pass the object and where I am meant to be using it. do I need to put it in a specially named file? I am a bit confused about this.

Please can someone put me through on this. Any help will be appreciated

**Comment by dcastil** [comment]
Hey @DevYemi! 👋

Did you manage to figure it out or do you still need help?

**Comment by DevYemi** [comment]
@dcastil Yeah, i was able to figure it out. Thank you so much
